packages/core/src/sass/app/_common/components/account-common.scss

```diff
@@ -301,15 +320,31 @@
             border-top: 1px solid var(--general-section-1);
             background-color: var(--general-main-1);
 
+            @include desktop-screen {
+                max-width: 100%;
+            }
+
+            @include mobile-or-tablet-screen {
+                flex-direction: row;
+                justify-content: center;
+                width: 100%;
+                left: 0;
```

Comment:
we can use logical property here for `left`

Reply:
done

Reply:
done